advocacy_docs/edb-postgres-ai/console/estate/monitor_aws.mdx

```diff
@@ -0,0 +1,121 @@
+
+Using this role ARN and a custom policy, the EDB Postgres AI server will have access to the RDS and S3 information in your AWS account. 
+
+After configuring the wizard with the role ARN, you will see the selected AWS resources (RDS instances and/or S3 buckets) in the chosen AWS regions on your **Estate** page in the **Cloud Hosted Databases** section.
+
+## Setting up monitoring of AWS resources in EDB Postgres AI Estate
+
+### Starting the Cloud Hosted Databases wizard
+
+1. Log into **EDB Postgres AI Console**. 
+
+2. Scroll down to the **Cloud Hosted Databases** section, select the **Manage Access** button, and choose your project.
+
+3. The **Cloud Hosted Databases** wizard shows **Step 1 - Create custom policy**
+
+### Creating the AWS custom policy
+
+4. Log into your AWS account with the RDS instances and S3 buckets you want to monitor.
+
+5. Navigate to IAM, and in the navigation pane on the left side of the AWS console, select **Policies**.
+
+6. On the **Policies** dashboard page, select the **Create policy**. button
+
+7. In the **Policy editor** section, choose the JSON option.
+
+8. Type or paste the following JSON policy document into the JSON editor:
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "VisualEditor0",
+      "Effect": "Allow",
+      "Action": [
+        "rds:DescribeDBInstances",
+        "s3:ListAllMyBuckets",
+        "rds:DescribeDBClusters"
+      ],
+      "Resource": "*"
+    }
+  ]
+}
+```
+
+9. Give the policy a name, for instance, **biganimal-beacon-addon-policy**. This policy allows EDB Postgres AI server to query metadata of your AWS RDS and S3 services.
+
+
+### Creating the AWS role
+
+10. In the wizard, select the **Next:Create a Role** button. The wizard should now show **Step 2 - Create a Role**.
+
+11. In the left-hand navigation pane of the AWS console, choose **Roles** and then select the **Create role** button.
+
+12. Select **Custom trust policy** role type.
+
+13. In the **Custom trust policy** section, paste the following trust policy for the role:
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "arn:aws:iam::292478331082:root"
+      },
+      "Action": "sts:AssumeRole",
+      "Condition": {
+        "StringEquals": {
+          "sts:ExternalId": "<project-id>"
+        }
+      }
+    }
+  ]
+}
+```
+
+!!! Note
+The EDB Postgres AI Cloud Hosted Databases wizard shows a snippet like the one above but with the `<project-id>` already specified.
+!!!
+
+14. Select the **Next** button.
+
+15. Select the policy you created earlier. In this example, we used **biganimal-beacon-addon-policy**.
```

Comment:
If you change the name above, a reminder to change it here as well. 
Just saying this because I usually forget things that are not explicitly commented 🙈

Reply:
Got it. Thanks. 👍